packages/components/src/internal/util/utils.ts

```diff
@@ -523,6 +523,10 @@ export function downloadAttachment(href: string, openInTab?: boolean, fileName?:
     }
 }
 
+export function downloadFromUrl(href: string): void {
+    window.open(href, "_blank", "noopener,noreferrer");
```

Comment:
should line 517 above from `downloadAttachment` call this? 

actually, should you just add the `"noopener,noreferrer"` part to line 517 and then call `downloadAttachment` for your use cases instead of adding a new helper?

Reply:
Good idea.  Done.